README.md

```diff
@@ -1,77 +1,16 @@
-# Gardener Website Content Repository
+# Gardener Documentation
 
-[![reuse compliant](https://reuse.software/badge/reuse-compliant.svg)](https://reuse.software/)
+## Structure
 
-## Website repositories and CI/CD Overview
+### Manifest files
 
-<img style="width:50%" src="images/overview.svg">
+The manifest files can be found in [.docforge](https://github.com/gardener/documentation/tree/master/.docforge) folder and they describe how the documentation will be mapped in the [https://gardener.cloud/](https://gardener.cloud/) website. The main manifest file which is the entrypoint of the website build is [website.yaml](https://github.com/gardener/documentation/blob/master/.docforge/website.yaml). It links content and other manifest files in this repository. 
+At the last level of linking, it can be seen that the manifests point to manifests from other repositories in the gardener organization. Example for such a manifest is [documentation.yaml](https://github.com/gardener/documentation/blob/master/.docforge/documentation/documentation.yaml).
```

Comment:
```suggestion
At the last level of linking, it can be seen that the manifests point to manifests from other repositories in the gardener organization. An example of such a manifest is [documentation.yaml](https://github.com/gardener/documentation/blob/master/.docforge/documentation/documentation.yaml).
```